back/node_watcher/src/nodeWatcher.ts

```diff
@@ -9,6 +9,8 @@ import { logger } from '@polkadot/util';
 import { NomidotTask } from './tasks/types';
 
 const ARCHIVE_NODE_ENDPOINT = 'wss://kusama-rpc.polkadot.io/';
+// const ARCHIVE_NODE_ENDPOINT = 'ws://127.0.0.1:9944';
```

Comment:
I would prefer not to have commented code in the repo (exactly as in substrate repo).

Even a comment "Replace with 'ws://127.0.0.1:9944'" would do

Reply:
```suggestion
```